### DISCUSSION TITLE
I wonder if it will interfere with prospective pull requests if I add a .github actions job to do the unit testing code?

### DISCUSSION THREAD

**Discussion by russelljjarvis** [SpikingNetwork/TrainSpikingNet.jl#6] [Ideas]
Will it interfere with prospective pull requests if I add a .github actions job to do the unit testing code?

I was thinking of recasting my connectome work as a unit test, .github actions unit test, but I also think this could step on toes, so I will hold back.

Also, do you prefer squashed commits for PR? I can try to squash everything into three commits.

**Comment by bjarthur** [comment]
definitely prefer squashed commits.

a github action to do CI is pretty standard, though i haven't gotten that fancy here.  if you decide to code that up, put it in a separate PR as that is entirely unrelated to potjans.

**Comment by bjarthur** [comment]
fixed by https://github.com/SpikingNetwork/TrainSpikingNet.jl/commit/974f679664bd0cab69eb6d35f4b5ee285d5d312e
